admin-manual/customization/authentication.rst

```diff
@@ -110,9 +114,234 @@ that are properly configured with an ``attribute_value`` and a valid
+will be globally unique, unlikely to change, and used to identify individual 
+users. If you want the login experience to match AtoM's current expectations 
+(using an email address in the username field) and your LDAP server is configured
+to use **mail** attributes for every user, then you would enter ``mail`` into
+this field. 
+
+If ``mail`` is not used in your LDAP server, ``uid`` is a good option, as this
+typically shouldn't change even if the entry moves in the LDAP directory tree. 
+Barring that, ``cn`` could be used. Whatever is picked will define what type of
+information end users attempting to authenticate in AtoM should enter into the 
+login panel's "Email" field. 
+
+.. image:: images/ldap-config-example.*
+   :align: center
+   :width: 90%
+   :alt: An image of example valudes in the LDAP authentication settings
+
+In the above example, users would enter their LDAP username into AtoM's
+"Email" field in the login menu. If ``cn`` were used as the lookup attribute
+instead, then users would need to enter their LDAP common name.
+
+After configuring the LDAP settings
+-----------------------------------
+
+Once configured, users registered on the LDAP server should now be able to
+authenticate with their LDAP credentials to log into AtoM. 
+
+When a user authenticates for the first time, a new AtoM user is created in the 
+database. However, by default, this user is **not assigned to any user groups**,
+and will therefore only have the base permissions of the Authenticated user 
+group. An :term:`administrator` can modify the user permissions after the 
+new user has been created using AtoM's permissions module - for more information, 
+see: 
+
+* :ref:`edit-user-permissions`
+* :ref:`manage-user-accounts`
+
+.. IMPORTANT:: 
+
+   If switching to LDAP authentication *after* creating and configuring local
+   user accounts in AtoM, the most secure method to transition will likely be to 
+   delete the local user accounts and let the LDAP authentication create new 
+   ones, so that no passwords are stored in AtoM and no duplicate accounts exist. 
+
+   When AtoM attempts to authenticate an LDAP user, it will: 
+
+   * Fall back and check if a local account matches the user credentials if no
+     LDAP match is found. This can allow users with existing local accounts to 
+     bypass the LDAP authentication, and/or continue accessing AtoM even if the 
+     LDAP entry is removed.
+   * If a match IS found, AtoM will attempt to look for existing accounts by
+     comparing the LDAP lookup attribute (user name, etc) against the emails
+     stored in AtoM's user table (which are used for local authentication).
+     This means that if the lookup attribute configured is **not** ``mail``,
+     then a new account could be created, leaving the user with both a local
+     user account and a newly created user account.
+   * If no local account match is found, AtoM will create a new default user, 
```

Comment:
This is the standard behaviour when a local account is not present. Should this be at the top if this is a list of reasons to not have a local account set up?